commitizen-tools/commitizen
# fix(cz_customize): make schema_pattern customizable through config for cz_customize

## Description
make schema_pattern customizable through config for cz_customize

## Checklist


## Expected behavior
configure `schema_pattern` in the configuration file should make `cz check` work for cz_customize

## Steps to Test This Pull Request
1. `cz init` (Choose `cz_customize`)
2. add `schema_pattern` to configuration file
3. run `cz check`

## Additional context
#273
